src/components/Layout/Navigation.css

```diff
@@ -10,7 +10,7 @@
   list-style-type: none;
 }
 
-.Navigation .dcl.tab svg {
+.dcl.tab svg {
```

Comment:
If it is not specific to Navigation anymore, please move it to the `ui-overrides` css file.